fab/target_distributions/many_well.py

```diff
@@ -96,22 +96,23 @@ def log_prob_2D(self, x):
     def performance_metrics(self, samples: torch.Tensor, log_w: torch.Tensor,
                             log_q_fn: Optional[LogProbFunc] = None,
                             batch_size: Optional[int] = None) -> Dict:
-        if log_q_fn is None:
```

Comment:
If `log_q_fn` is None we can still evaluate the `log_w` accuracy in estimating the normalizing constant.